src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Make random an instance level variable to not create a new one each time
`Color.values()` call result can be saved to a variable to avoid multiple calls